sdk/python/sawtooth_sdk/workload/workload_generator.py

```diff
@@ -73,64 +73,69 @@ def run(self):
 
     @asyncio.coroutine
     def _simulator_loop(self):
+        while True:
+            yield from asyncio.sleep(self._rate)
+            with self._lock:
```

Comment:
Looks like we're mixing async io event loops and threading locks - this will block the whole event loop, as we saw in the rest api case (#303).  This should probably be an `asyncio.Lock`